resolver_wrapper.go

```diff
@@ -187,7 +187,7 @@ func (ccr *ccResolverWrapper) ParseServiceConfig(scJSON string) *serviceconfig.P
 // addChannelzTraceEvent adds a channelz trace event containing the new
 // state received from resolver implementations.
 func (ccr *ccResolverWrapper) addChannelzTraceEvent(s resolver.State) {
-	if !logger.V(0) && !channelz.IsOn() {
+	if !logger.V(2) || !channelz.IsOn() {
```

Comment:
We don't want the verbosity check to affect whether or not channelz traces are being added. The `channelz.Infof` adds a log and a trace.

I would be OK with removing the verbosity check completely and only checking if channelz is turned on.

Reply:
Ah, I just read the PR description and noticed that there is a change from `&&` to `||`. Even with that change, I still think that totally getting rid of the verbosity check would be better, since it would retain existing behavior instead of changing it.